blocks/lead-art-block/themes/news.json

```diff
@@ -33,7 +33,7 @@
 						"display": "none"
```

Comment:
Github isn't letting comment on the exact line, but on line 13 above there's two 0's without a space: `"padding": "00 var(--global-spacing-2) 0"`. It should be `"padding": "0 0 var(--global-spacing-2) 0"`. We should look for similar issues in the json files and fix them. The only way I can think this happened in the first place was the extension that converted the JS to JSON mishandled these strings.

Reply:
Thanks @vgalatro, Fixed it in [ddaa22c](https://github.com/WPMedia/arc-themes-blocks/pull/1643/commits/ddaa22c42228792bef6e603144ecbe48b82123a2). Let me know if any more changes needed.